quickstarts/scanned-events-query-governance/config.yml

```diff
@@ -0,0 +1,45 @@
+id: 
```

Comment:
Hey @caromedellin, let's remove this line, it will be automatically added once merged.
```suggestion
```

Reply:
Hey @caromedellin! I would just push everything up to this branch, that keeps things together.
The data source needs to exist either here or in our repo of CORE data sources otherwise this quickstart will not be installable on a user's account. Overall, we can't do a limited release of a data source because then the quickstart would break for all the users that are not part of that limited release.